gravatar/src/main/java/com/gravatar/services/ServiceUtils.kt

```diff
@@ -3,6 +3,11 @@ package com.gravatar.services
 import kotlinx.coroutines.CancellationException
 import com.gravatar.di.container.GravatarSdkContainer.Companion.instance as GravatarSdkDI
 
+@Deprecated(
+    "This method is deprecated and should be removed in a future release.",
+    replaceWith = ReplaceWith("com.gravatar.services.runCatchingRequest"),
+    level = DeprecationLevel.WARNING,
+)
 internal inline fun <T> runCatchingService(block: () -> Result<T, ErrorType>): Result<T, ErrorType> {
```

Comment:
This is internal - do we have to deprecate it? Can't we remove it?

Reply:
I didn't remove it because is still in use [here](https://github.com/Automattic/Gravatar-SDK-Android/blob/1f754d9e778df05266e9810d432cc26695f44c07/gravatar/src/main/java/com/gravatar/services/ProfileService.kt#L34) so I think we can remove it when we remove the other deprecated methods.